pupa/importers/vote_events.py

```diff
@@ -70,8 +69,9 @@ def prepare_for_db(self, data):
 
         bill = data.pop('bill')
         if bill and bill.startswith('~'):
+            # unpack psuedo id and apply filter in case there are any that alter it
```

Comment:
filters need to be applied at import time to handle these kinds of cases (more will likely arise as we enable filters elsewhere)